content/account/app/stats.textile

```diff
@@ -0,0 +1,51 @@
+---
+title: Stats
+meta_description: “Monitor and analyze your app's performance with Ably's dashboard. Access realtime stats and trends for optimized management."
+meta_keywords: "Ably dashboard, dashboard, app performance, realtime stats, usage monitoring, performance analysis, data trends, statistics chart, statistics table"
+---
+
+The stats tab is an interface to monitor your app's performance and usage via the "statistics table":#table and "statistics chart":#chart.
+
+h2(#table). Statistics table
+
+The statistics table provides a summary of your app's messaging and data usage patterns over different time frames, including the previous month, the current month, and more granular insights from the last hour and last minute:
+
+<a href="@content/screenshots/dash/stats-table.png" target="_blank">
+  <img src="@content/screenshots/dash/stats-table.png" style="width: 100%" alt="Your stats table">
+</a>
+
+The following explains the statistics table metrics:
+
+|_. Metric |_. Description |
+| *Messages (billable)* | Number of messages that were billable. |
+| *Messages published (REST & Realtime)* | Number of messages sent in REST & Realtime. |
```

Comment:
Any of these with REST and realtime should be "via" rather than "in". We also don't use `&`, so it's correct in the label title because it matches the UI, but we should use `and` in the description.